caddyhttp/errors/errors.go

```diff
@@ -83,9 +85,13 @@ func (h ErrorHandler) errorPage(w http.ResponseWriter, r *http.Request, code int
 			return
 		}
 		defer errorPage.Close()
-
+		// Get content type by extension
+		contentType := mime.TypeByExtension(filepath.Ext(pagePath))
+		if contentType == ""{
```

Comment:
Please run `go fmt` on all modified files

Reply:
Hi @mholt I have solved this problem.